### DISCUSSION TITLE
how to add fira code glyphs to other fontz

### DISCUSSION THREAD

**Discussion by bryant-the-coder** [tonsky/FiraCode#1376] [Q&A]
the title says it all.

**Comment by tonsky** [comment]
It might be tricky. You’ll probably need a font editor, then copy & paste the glyphs and the calt code

**Reply by bryant-the-coder** [reply]
how? isnt there a simpler way?

**Reply by tonsky** [reply]
How what?

**Reply by bryant-the-coder** [reply]
what font editor? how to copy and paste? There is no script for it?

**Reply by tonsky** [reply]
No script. You’ll need to do your own research on it. I am using Glyphs but it might be an overkill for you as it’s not free

**Reply by bryant-the-coder** [reply]
:(

**Comment by Daxtorim** [comment]
[EzequielRamis/liga](https://github.com/EzequielRamis/liga) can do that.

**Reply by bryant-the-coder** [reply]
i m on windows btw
